vispy/visuals/visual.py

```diff
@@ -649,7 +649,7 @@ def _compute_bounds(self, axis, view):
                 vb = v.bounds(axis)
                 if bounds is None:
                     bounds = vb
-                else:
+                elif vb is not None:
```

Comment:
@campagnola: Can you could confirm that this is a bug and not intended behavior?